docs/3.0.0-beta.x/getting-started/quick-start-tutorial.md

```diff
@@ -577,7 +578,7 @@ You have now entered your first **Restaurant** Content Type. You have also assig
 
 ## 7. Set Roles and Permissions
 
-By default, Strapi publishes all **Content Types** with restricted permissions. Which means you have to explicitly give permissions to each **Content Type** you create. You are going to give **Public** API (or URL) access to both the **Restaurant** Content Type and **Category** Content Type.
+By default, Strapi publishes all **Collection Types** with restricted permissions. Which means you have to explicitly give permissions to each **Collection Type** you create. You are going to give **Public** API (or URL) access to both the **Restaurant** Collection Type and **Category** Collection Type.
```

Comment:
Here you have to use `Content Type` for the 2 first.
Both Collection and Single Types have restrictions.

By default, Strapi publishes all **Content Types** with restricted permissions. This means you have to explicitly give permissions to each **Content Type** you create. You are going to give **Public** API (or URL) access to both the **Restaurant** Collection Type and **Category** Collection Type.